### PR TITLE
fix: :bug: Fix error when responding with paginator to an ephemeral deferred interaction

### PR DESCRIPTION
**Pull request by Paillat-dev**
## Summary
fix: #2173 

## Information

- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2624](https://github.com/Pycord-Development/pycord/pull/2624))
 - Fixed `AttributeError` when accessing `Member.guild_permissions` for user installed
   apps. ([#2650](https://github.com/Pycord-Development/pycord/pull/2650))
+- Fixed error when responding normally with a `Paginator` to an ephemerally deferred
```
Comment (JustaSqu1d) [inline CHANGELOG.md:65]:
```suggestion
- Fixed error when responding with a `Paginator` to an ephemerally deferred
```

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Paillat-dev) [reply CHANGELOG.md]: @JustaSqu1d Is it good like now ?

Comment (JustaSqu1d) [reply CHANGELOG.md]: "Normally" seems redundant

Comment (Paillat-dev) [reply CHANGELOG.md]: Yeah no I removed the normally.

Comment (Paillat-dev) [reply CHANGELOG.md]: @JustaSqu1d ?

**Review by JustaSqu1d — APPROVED** [review]

**Review by Dorukyum — APPROVED** [review]

**Pull request merged**